### PR TITLE
Update docs inline with the new custom element integration

### PR DESCRIPTION
**Pull request by errm**

### CODE REVIEW

File: docs/reevoomark/conversation-badges.markdown [hunk 1]
```diff
@@ -13,18 +13,7 @@ next: tracking
 To display the conversation badge on your website you have to
 
 1. [include the Reevoomark Javascript library](../javascript-library)
```
Comment (jonnyarnold) [inline docs/reevoomark/conversation-badges.markdown]: Can we get capitals at the start of these lines?

File: docs/reevoomark/ie8.markdown [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+{% highlight html %}
+<head>
+  ...
+  <!--[if lte IE 8]><script src="//cdn.mark.reevoo.com/assets/ie8.js"></script><![endif]-->
+</head>
+{% endhighlight %}
+
+It is important that this script is loaded into the page synchronously before
+rendering begins in IE8 or we will not be able to display badges.
```
Comment (jonnyarnold) [inline docs/reevoomark/ie8.markdown]: Is this a shim? Have we isolated this to ensure we only shim our own stuff, and not the client's?

**Comment by jonnyarnold** [comment]
LGTM following comments, once new badges are deployed.

File: docs/reevoomark/ie8.markdown (hunk 2 above)
Comment (reneklacan) [inline docs/reevoomark/ie8.markdown]: @errm ?

Comment (errm) [inline docs/reevoomark/ie8.markdown]: It needs to shimmed globally, as we don't know how or where badges will be created by the client.

**Comment by jonnyarnold** [comment]
Release date will be confirmed once we have talked with CX.

**Comment by jonnyarnold** [comment]
@JohnPlummer Do we have a release date planned for this?

**Comment by jonnyarnold** [comment]
Rebased onto master and ready to go!

**Pull request merged**